packages/server/src/database/models.ts

```diff
@@ -5,37 +5,50 @@ import { GameState } from 'shared';
 export interface IPlayer extends mongoose.Document {
   name: string;
 }
-const PlayerSchema = new mongoose.Schema({
-  name: String,
-});
+const PlayerSchema = new mongoose.Schema(
+  {
+    name: String,
+    createdAt: Date,
```

Comment:
Are you sure the `createdAt` and `updateAt` are needed here? I think according to documentation, mongoose will add those. Maybe we need them in the interface, but not the schema.
https://mongoosejs.com/docs/guide.html#timestamps

Reply:
I'm not :-P how could I test this? Should I just remove them and see if everything works?

Reply:
done